charts/connect/templates/NOTES.txt

```diff
@@ -45,15 +45,27 @@
 ---------------------------------------------------------------------------------------------
 {{- end }}
 
+{{- if and (not (.Values.connect.create)) ( not (.Values.operator.create)) }}
+  {{- fail "The `connect.create` and `operator.create` are both set to `false`. At least one should be set to `true`." -}}
+{{- end -}}
+
 ** Please be patient while the chart is being deployed **
 
 1Password Connect is being deployed to Kubernetes. More information about 1Password Connect can 
 be found at https://support.1password.com/secrets-automation/
 
-{{- if  .Values.operator.create }}
+{{- if (.Values.connect.create) and (.Values.operator.create) }}
```

Comment:
I think this should be
`{{- if and (.Values.connect.create) (.Values.operator.create) }}`
instead, i.e. prefixed and instead of infix?

Likewise, below should probably be `{{- if and (not (.Values.connect.create)) (.Values.operator.create) }}`?

I'm basing this on https://helm.sh/docs/chart_template_guide/function_list/#and

I'm guessing this is what causing your linter to complain.